docs/CommandInput.md

```diff
@@ -12,7 +12,7 @@ The executable can take in inputs defined in `config.dev.json`, or as command li
 | OutputDirectory | Path to folder the test results will be placed. Optional. If this is not provided, it will be placed in the `TestOutput` folder. |
 | LogLevel | Level for logging (Folllows [this](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=dotnet-plat-ext-6.0)). Optional. If this is not provided, Information level logs and higher will be logged |
 | QueryParams | Specify query parameters to be added to the Power Apps URL. |
-
+| Domain | Specify what URL domain your app uses. |
```

Comment:
(1) This should default to the public domain if not specified, (2) the docs should note here that it's optional and what the default value is